### PR TITLE
Feature/kitosudv 2219 task 2319

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.ApplicationServices/Model/Contracts/Write/ItContractModificationParameters.cs [hunk 1]
```diff
@@ -7,11 +7,9 @@ namespace Core.ApplicationServices.Model.Contracts.Write
...
     {
         public OptionalValueChange<string> Name { get; set; } = OptionalValueChange<string>.None;
-		
-		public OptionalValueChange<Guid?> ParentContractUuid { get; set; } = OptionalValueChange<Guid?>.None;
-
+        public OptionalValueChange<Guid?> ParentContractUuid { get; set; } = OptionalValueChange<Guid?>.None;
         public Maybe<ItContractGeneralDataModificationParameters> General { get; set; } = Maybe<ItContractGeneralDataModificationParameters>.None;
-
-        public Maybe<ItContractProcurementModificationParameters> Procurement { get; set; } = Maybe<ItContractProcurementModificationParameters>.None;
+		public Maybe<ItContractProcurementModificationParameters> Procurement { get; set; } = Maybe<ItContractProcurementModificationParameters>.None;
```
Comment (JacobMalling) [inline Core.ApplicationServices/Model/Contracts/Write/ItContractModificationParameters.cs:12]: spacing

File: Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/IItContractWriteModelMapper.cs [hunk 2]
```diff
@@ -9,5 +9,6 @@ public interface IItContractWriteModelMapper
         ItContractModificationParameters FromPUT(UpdateContractRequestDTO dto);
         ItContractGeneralDataModificationParameters MapGeneralData(ContractGeneralDataWriteRequestDTO generalDataDto);
         ItContractProcurementModificationParameters MapProcurement(ContractProcurementDataWriteRequestDTO request);
+		ItContractResponsibleDataModificationParameters MapResponsible(ContractResponsibleDataWriteRequestDTO dto);
```
Comment (JacobMalling) [inline Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/IItContractWriteModelMapper.cs:12]: spacing

File: Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs [hunk 3]
```diff
@@ -162,6 +165,54 @@ public void Can_Map_Parent_From_Put(bool hasParentUuid)
...
+        }
+
+        private static void AssertResponsible(ContractResponsibleDataWriteRequestDTO input,
+            ItContractResponsibleDataModificationParameters output)
+        {
+            Assert.Equal(input.OrganizationUnitUuid, AssertPropertyContainsDataChange(output.OrganizationUnitUuid));
+            Assert.Equal(input.Signed, AssertPropertyContainsDataChange(output.Signed));
+            Assert.Equal(input.SignedAt, AssertPropertyContainsDataChange(output.SignedAt));
+            Assert.Equal(input.SignedBy, AssertPropertyContainsDataChange(output.SignedBy));
+		}
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs:214]: spacing

**Review by JacobMalling — APPROVED** [review]
Lidt underligt spacing hist og her. Ligner noget formatering der er gået galt hvor det nogle steder er tabs og andre steder er spaces

File: Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs [hunk 4]
```diff
@@ -762,6 +762,98 @@ public async Task Can_PUT_With_GeneralData()
...
+
+            var request = new CreateNewContractRequestDTO
+            {
+                OrganizationUuid = organization.Uuid,
+                Name = CreateName(),
+            };
+            var dto = await ItContractV2Helper.PostContractAsync(token, request);
+
+            //Act
+            var changes = await CreateContractResponsibleDataRequestDTO(token, organization, false, false, false);
```
[the remark this answers is not in the document]
Comment (mrjsawdk) [reply Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs:807]: god ide

**Pull request merged**